extensions/amp-story/1.0/amp-story-reaction.js

```diff
@@ -225,47 +257,30 @@ export class AmpStoryReaction extends AMP.BaseElement {
       true /** callToInitialize */
     );
 
-    // Add a click listener to the element to trigger the class change
-    this.rootEl_.addEventListener('click', (e) => this.handleTap_(e));
-  }
-
-  /**
-   * Handles a tap event on the quiz element.
-   * @param {Event} e
-   * @private
-   */
-  handleTap_(e) {
-    if (this.hasUserSelection_) {
-      return;
-    }
-
-    const optionEl = closest(
-      dev().assertElement(e.target),
-      (element) => {
-        return element.classList.contains('i-amphtml-story-reaction-option');
-      },
-      this.rootEl_
+    this.optionElements_ = toArray(
+      this.rootEl_.querySelectorAll('.i-amphtml-story-reaction-option')
     );
 
-    if (optionEl) {
-      this.handleOptionSelection_(optionEl);
-    }
+    this.optionElements_.forEach((element, reactionValue) => {
```

Comment:
This was the original way of calling the option selected (index), but I can change the naming of it everywhere.

Reply:
Changed to optionSelected when the context implies that the variable refers to the index of the selected option, and changed to optionIndex when it iterates over all the indices, hopefully it will be more clear now.